ex2/app/src/main/java/com/example/app_v3/HimymAdapter.java

```diff
@@ -0,0 +1,58 @@
+package com.example.app_v3;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import org.w3c.dom.Text;
+
+import java.util.List;
+
+public class HimymAdapter extends RecyclerView.Adapter<HimymAdapter.HimymViewHolder> {
+    private List<String> hList;
```

Comment:
Move intuitive are better, what does hList stands for?

Reply:
h stands for **H**ow I met your mother, it's too long...
I'll try to make it more intuitive next time :)